src/client/tst/StreamPutGetTest.cpp

```diff
@@ -1518,7 +1518,7 @@ TEST_F(StreamPutGetTest, DISABLED_putFrame_PutGetTagsBeforeStoreData)
     ASSERT_TRUE(retStatus == STATUS_SUCCESS || retStatus == STATUS_NO_MORE_DATA_AVAILABLE);
 
     // Manually pre-validated data file size
-    EXPECT_EQ(6700, filledSize);
+    EXPECT_EQ(6703, filledSize);
```

Comment:
Can we have a #define for 6703? It looks like a magic number and having a #define will be useful for understanding in the future.

Reply:
This would have been absolutely a no go for the codebase but the tests assume these numbers have been sanitized by hand (which we do by running mkvgen -v). As the test code is peppered with these, I won't go and change these now - we can scrub them at a later stage if needed.